sched/scheduler/jobStatus.go

```diff
@@ -0,0 +1,83 @@
+package scheduler
+
+import (
+	s "github.com/scootdev/scoot/saga"
+	"github.com/scootdev/scoot/sched"
+)
+
+type JobStatus struct {
+	Id         string
+	Status     sched.Status
```

Comment:
It always seems weird to me to have a struct with Status in the name with an enum that's named Status. Does this bug you? How can we